### PR TITLE
MacOS: core-graphics update to v12

### PR DESCRIPTION
**Pull request by NatalyaKovalova**
Webrender was updated to Core Graphics v12.
It is required for iOS support.
[Core-graphics's PR should be approved before this](https://github.com/servo/core-graphics-rs/pull/106)

Some Servos code should be changed for this version of webrender( [gfx component](https://github.com/servo/servo/blob/master/components/gfx/platform/macos/font_template.rs), in `pub fn ctfont(&self, pt_size: f64) -> Option<CTFont>` ), should I up version in Webrender's cargo?

@larsbergstrom

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/2056)

### CODE REVIEW

File: webrender/src/platform/macos/font.rs [hunk 1]
```diff
@@ -21,6 +21,8 @@ use core_graphics::geometry::{CGPoint, CGRect, CGSize};
 use core_text;
 use core_text::font::{CTFont, CTFontRef};
 use core_text::font_descriptor::{kCTFontDefaultOrientation, kCTFontColorGlyphsTrait};
+use foreign_types::ForeignType;
+use foreign_types::ForeignTypeRef;
```
Comment (kvark) [inline webrender/src/platform/macos/font.rs]: nit: merge into a single `use` statement?

File: webrender/src/platform/macos/font.rs [hunk 2]
```diff
@@ -244,9 +244,11 @@ fn new_ct_font_with_variations(cg_font: &CGFont, size: Au, variations: &[FontVar
             return ct_font;
         }
         let vals_dict = CFDictionary::from_CFType_pairs(&vals);
-        let cg_var_font_ref = CGFontCreateCopyWithVariations(cg_font.as_concrete_TypeRef(), vals_dict.as_concrete_TypeRef());
-        let cg_var_font: CGFont = TCFType::wrap_under_create_rule(cg_var_font_ref);
-        core_text::font::new_from_CGFont(&cg_var_font, size.to_f64_px())
+        let cg_font = cg_font.create_copy_from_variations( &vals_dict );
+        match cg_font {
+                Ok(f) => return core_text::font::new_from_CGFont(&f, size.to_f64_px()),
```
Comment (kvark) [inline webrender/src/platform/macos/font.rs]: no returns are needed in this `match` arms

**Comment by glennw** [comment]
I assume this can be closed as a duplicate of https://github.com/servo/webrender/pull/2063/files ?

**Comment by glennw** [comment]
Closing this since I think it's superseded by #2063 - please feel free to re-open if still needed.

**Pull request closed without merging**